### DISCUSSION TITLE
Privacy question

### DISCUSSION THREAD

**Discussion by Naxterra** [simple-login/app#1077] [Q&A] [answered]
Hello

I have a question related to privacy. 

Let's say I created several aliases, used them in public places like forums, FB, Twitter, and so on. If a legal entity or government ask for details of the email alias, will Simple Login provide all the details to the requester? For example, I created an email alias, used in forums, and some government official didn't like my post and took it to the court. What will happen? Will SL provide original email owner details to the requester as well as emails forwarded under that alias, or SL will say that is not our problem, we have no clue who is the owner?

Also, even if I create my account with privacy based email service like Proton, how can I make sure that my identity remains anonymous? Because at some point I have to provide payment details for the Premium service, and I don't know if SL will give that info to the authorities or not.

**Comment by nguyenkims** [comment, marked as answer]
Legally SimpleLogin has to comply with French law. SimpleLogin will need to provide the info we have on the alias, including the mailbox that receives emails sent to it. We don't store email content.
Your payment is linked to your account so we need to provide that info as well. You can use crypto as payment for anonymity.

**Reply by Redsandro** [reply]
Will the Proton acquisition change anything about that in the future? (E.g. comply with Swiss law in stead of French law.)

**Reply by c0nfigurati0n** [reply]
@Redsandro It shouldn't, because SimpleLogin is still it's own separate legal entity which has to comply with the laws and regulations of where said separate legal entity is registered at (the legal jurisdiction of the country its located in that is). .

**Comment by seanPhill** [comment]
So if the receiving mailbox is a Protonmail one, all they'll get is your Protonmail email address.

**Reply by c0nfigurati0n** [reply]
That should be the case (i think), unless you have multiple email inboxes located at different email services?!
